lib/dialects/sqlite/query-generator.js

```diff
@@ -79,8 +82,139 @@ const QueryGenerator = {
     return !!value ? 1 : 0;
   },
 
-  addColumnQuery(table, key, dataType) {
+  handleSequelizeMethod(smth, tableName, factory, options, prepend) {
+    if (smth instanceof Utils.Json) {
+      // Parse nested object
+      if (smth.conditions) {
+        const conditions = _.map(this.parseConditionObject(smth.conditions), condition =>
+          `json_extract(${this.quoteIdentifier(_.first(condition.path))}, '$.${_.tail(condition.path).join('.')}') = '${condition.value}'`
+        );
+
+        return conditions.join(' and ');
+      } else if (smth.path) {
+        let str;
+
+        // Allow specifying conditions using the sqlite json1 functions
+        // https://sqlite.org/json1.html
+        if (/(json(?:_.*)*)\(.*\)/i.test(smth.path)) {
```

Comment:
@rokoroku Could you please add a case for SQL Injection attack (please check [these tests](https://github.com/sequelize/sequelize/blob/5b653701849325b298df7b27e7fdf1f8c273f1ec/test/integration/model/geometry.test.js#L181) for inspiration)